app/src/main/java/org/systers/mentorship/remote/requests/Register.kt

```diff
@@ -19,4 +19,5 @@ data class Register (
         val password: String,
         val termsAndConditionsChecked: Boolean,
         val needMentoring: Boolean,
-        val availableToMentor: Boolean)
+        val availableToMentor: Boolean,
+        val isAvailableForBoth: Boolean)
```

Comment:
you don't need this extra parameter, the backend is not expecting it. so you would need to change the backend for it. Here what you can do, is have some logical condition, that if the user selects "Both" option in the UI, then you have to set both `needMentoring` and `availableToMentor` to True :) Makes sense?

For reference, here's what the backend waits as the request:
![image](https://user-images.githubusercontent.com/11148726/79871892-f8e4a800-83dc-11ea-8750-ad130a2accb8.png)

Reply:
@isabelcosta  Oh right. Yeah I ll do the same. Thanks :)